### ISSUE TITLE
CREPE model Tensorflow on Android

### ISSUE THREAD

**Issue by remyut** [marl/crepe#79]
Hi,

I use the CREPE model on a web browser which works pretty fine, but is there a way to integrate it in Android or flutter web/mobile using Tensorflow library?

the model works just nice when the mic is used from the web browser (with ml5 js for example). I wish to connect another source, like phone mic, is this possible? 

Could you please help me to understand?

I believe it should be possible, I do not want to re-train the model it already works fine.

Thanks for your help
Remy

**Comment by martingasser** [comment]
I'm using it right now in an iOS/Android React Native app using TFLite.

You can read up on how to convert the Keras model to TFLite here: https://www.tensorflow.org/lite/convert

I just had to write a small conversion script that builds the Keras model and loads the weights from the h5 files provided in the repo, and then I used the tf.lite.TFLiteConverter.from_keras_model(...) API.

**Comment by remyut** [comment]
Thats great I’ll  check it out, thanks for the help

Remy

**Comment by martingasser** [comment]
> > I'm using it right now in an iOS/Android React Native app using TFLite.
> > You can read up on how to convert the Keras model to TFLite here: https://www.tensorflow.org/lite/convert
> > I just had to write a small conversion script that builds the Keras model and loads the weights from the h5 files provided in the repo, and then I used the tf.lite.TFLiteConverter.from_keras_model(...) API.
> 
> Hey, were you able to load a model bigger than the tiny one? The bigger model is the more latence I have during the inference

Of course, bigger models need more computation. On a low-end Android device (Samsung Galaxy A12), I can only run the tiny model in real time. On an iPhone 11, I can use the "small" model without problems. 

Which device are you using?

**Comment by Houvven** [comment]
> I'm using it right now in an iOS/Android React Native app using TFLite.
> 
> You can read up on how to convert the Keras model to TFLite here: https://www.tensorflow.org/lite/convert
> 
> I just had to write a small conversion script that builds the Keras model and loads the weights from the h5 files provided in the repo, and then I used the tf.lite.TFLiteConverter.from_keras_model(...) API.

hello, how can i make it to tflite? i m trying it, but get an error: No model config found in the file at ./model-small.h5.

**Comment by phamconganh** [comment]
@martingasser When converting, do you need to enable tf.lite.OpsSet.TFLITE_BUILTINS or tf.lite.OpsSet.SELECT_TF_OPS ?
